vermouth/oxDNA/oxDNA.py

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 University of Groningen
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Provides functionality to read and write oxDNA files.
+"""
+
+from itertools import chain
+
+import numpy as np
+
+from ..molecule import Molecule
+
+
+def read_oxDNA(file_name, exclude=()):
+    """
+    Parse a oxDNA file to create a molecule.
+
+    Parameters
+    ----------
+    filename: str
+        The file to read.
+    exclude: collections.abc.Container[str]
+        Atoms that have one of these residue names will not be included.
+
+    Returns
+    -------
+    vermouth.molecule.Molecule
+        The parsed molecules. Will not contain edges.
+    """
+    molecule = Molecule()
```

Comment:
The PDB parser returns a System, and I think that would also make more sense here?